docker-compose.yml

```diff
@@ -80,6 +80,24 @@ services:
     volumes:
       - ./public/packs:/mastodon/public/packs
       - ./public/system:/mastodon/public/system
+## Uncomment to enable federation with tor instances along with adding the following ENV variables
+## http_proxy=http://privoxy:8118
+## ALLOW_ACCESS_TO_HIDDEN_SERVICE=true
+## HIDDEN_SERVICE_VIA_TRANSPARENT_PROXY=true
```

Comment:
if `http_proxy` is used, `HIDDEN_SERVICE_VIA_TRANSPARENT_PROXY` is not used so it should be removed.